src/main/java/com/aventstack/extentreports/ExtentTest.java

```diff
@@ -1201,7 +1201,16 @@ public Status getStatus() {
     public Test getModel() {
         return test;
     }
-    
+
+    /**
+     * Convenience method for setting the description of the internal test model
+     *
+     * @param desc The description of the test case in the logs
+     */
+    public void setDescription(String desc) {
+        test.setDescription(desc);
```

Comment:
Can you do one thing here, please add some logic to test if the string is empty, null or trim it from preceding and proceeding spaces.
It will be better if you go ahead and move that logic inside the setDescription method instead of doing it on the caller side.